SIT.Manager.Avalonia/Views/MainWindow.axaml.cs

```diff
@@ -1,14 +1,27 @@
-﻿using Avalonia.Controls;
+﻿using FluentAvalonia.UI.Windowing;
 using Microsoft.Extensions.DependencyInjection;
 using SIT.Manager.Avalonia.ManagedProcess;
+using Avalonia.Media;
 using System;
 
 namespace SIT.Manager.Avalonia.Views;
 
-public partial class MainWindow : Window
+public partial class MainWindow : AppWindow
 {
     public MainWindow() {
         InitializeComponent();
+        TitleBar.BackgroundColor = new Color(0xFF, 0x00, 0x00, 0x00);
+        TitleBar.ForegroundColor = new Color(0xFF, 0xFF, 0xFF, 0xFF);
+        TitleBar.InactiveBackgroundColor = new Color(0xFF, 0x00, 0x00, 0x00);
+        TitleBar.InactiveForegroundColor = new Color(0xFF, 0xFF, 0xFF, 0xFF);
+        TitleBar.ButtonBackgroundColor = new Color(0xFF, 0x00, 0x00, 0x00);
+        TitleBar.ButtonForegroundColor = new Color(0xFF, 0xFF, 0xFF, 0xFF);
+        TitleBar.ButtonHoverBackgroundColor = new Color(0xFF, 0x11, 0x11, 0x11);
+        TitleBar.ButtonHoverForegroundColor = new Color(0xFF, 0xFF, 0xFF, 0xFF);
+        TitleBar.ButtonPressedBackgroundColor = new Color(0xFF, 0x21, 0x21, 0x21);
+        TitleBar.ButtonPressedForegroundColor = new Color(0xFF, 0xFF, 0xFF, 0xFF);
+        TitleBar.ButtonInactiveBackgroundColor = new Color(0xFF, 0x00, 0x00, 0x00);
+        TitleBar.ButtonInactiveForegroundColor = new Color(0xFF, 0xFF, 0xFF, 0xFF);
```

Comment:
Are these exposed in the xaml at all? I'd rather use xaml for this if possible

Reply:
Let me see...

Reply:
It is, but it's not recognizable by the intellisense. Let me try to fix that.

Reply:
No, just tested it doesn't want to grab those values from xaml. :( we have to use it only through source.

Reply:
Maybe, I am blind, here is the documentation.
https://amwx.github.io/FluentAvaloniaDocs/pages/Controls/AppWindow#related-enumsclasses

Reply:
No chance, by the way UWP has done it same way of accessing color elements. You cannot change them in xaml file, but you only can change them in initialization of the source code. So bad luck on this one.